packages/core/src/components/Typography/FontSize/EditFontSizeModal.tsx

```diff
@@ -60,15 +60,28 @@ export function EditFontSizeModal({
   )
 
   const [error, setError] = useState<string | null>(null)
+  const variantNameRef = useRef() as MutableRefObject<HTMLInputElement>
+  const variantValueRef = useRef() as MutableRefObject<HTMLInputElement>
 
   const handleSave = () => {
     setError(null)
-    if (variantName === '') {
+    if (variantName === '' || variantValue === '') {
+      // eslint-disable-next-line @typescript-eslint/no-unused-expressions
+      variantName === ''
+        ? variantNameRef.current.focus()
+        : variantValueRef.current.focus()
```

Comment:
```suggestion
    if (variantName !== '' && variantValue !== '') {
      return;
    } else if (variantName === '') {
      variantNameRef.current.focus();
    } else if (variantValue === '') {
      variantValueRef.current.focus()
    }
```

Reply:
Personal opinion 😅 , but I feel it's easier to read this way. Feel free to resolve the comment if you don't agree.